_assignments/mp4.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: assignment
+title: "MP4: Flight Simulator"
+parts:
+  -
+    title: MP4
+    due: March 25, 2019 @ 11:59 PM
+material:
+  -
+    name: Starting Unity package
+    url: https://courses.engr.illinois.edu/cs498vr3/fa2018/mps/MP4v2.unitypackage
+rubric:
+  -
+    name: Plane
+    points: 5
+    description: Game has a controllable plane
+  -
+    name: Acceleration/Deceleration
+    points: 5
+    description: Plane can accelerate and decelerate comfortably
+  -
+    name: Rotation
+    points: 10
+    description: TPlane can yaw, pitch, and roll.
+  -
+    name: Uses Touch Controllers
+    points: 25
+    description: Plane control scheme makes inventive use of rotation and position tracking of touch controllers
+  -
+    name: Weapon
+    points: 10
+    description: Pressing a button shoots out a raycast or bullet from the plane
+  -
+    name: Spheres
+    points: 5
+    description: There are spheres around the terrain
+  -
+    name: Shooting Game
+    points: 25
+    description: Player can shoot spheres, which disappear, then reappear
+  -
+    name: Tutorial
+    points: 10
+    description: Game has in-world tutorial that clearly communicates how to control the plane
+  -
+    name: Start Menu
+    points: 10
+    description: Game has a legible and understandable in-world start menu
+  -
+    name: Laser pointer
+    points: 20
+    description: The Start Menu is controlled by an in-game, grabbable laser pointer
+  -
+    name: Displaying Score
+    points: 5
+    description: Score is displayed in-world
+  -
+    name: Sound
+    points: 20
+    description: Elements of the simulator that should have sound, do.
+  -
+    name: Game over
+    points: 10
+    description: Game handles plane crashes in some manner
+  -
+    name: Framerate/Comfort
+    points: 20
+    description: Framerate rarely drops below 60 fps, game sound is not obnoxious, and plane acceleration, deceleration, and rotations are comfortable.
+  -
+    name: Interactive tutorial
+    points: 20 EC
+    description: The tutorial provided to the player is interactive
+  -
+
+points: 200
+---
+
+## Part 1 - Environment and Flight
+
+### Simulator Environment
+
+1. Create a new scene, and load the provided Unity package, which contains a large terrain for you to use. Read up on [Terrains in Unity](https://docs.unity3d.com/Manual/script-Terrain.html) 

```

Comment:
I'd _very gently_ (i.e. don't do this unless you're super bored) suggest putting single line breaks at the ends of each sentence (NOT DOUBLE as that will reflow the markdown). This makes it slightly easier to read in the web editor, which enforces line wraps that make the paragraphs look really odd.